File: Python-Assignment/Chapter-Three/Question11.java
import java.util.Scanner;

public class Question11{
   public static void main (String []args){

	Scanner input = new Scanner (System.in);

	double totalMiles = 0;
	double totalGallons = 0;

	System.out.print("Enter miles driven: ");
	double milesDriven = input.nextDouble();

	while (milesDriven != -1){
		totalMiles += milesDriven;
		
		System.out.print("Enter gallons used: ");
		double gallonsUsed = input.nextDouble();

		totalGallons += gallonsUsed;

		double milesPerGallon = milesDriven / gallonsUsed;

		System.out.println("The miles per gallon for this tank: " + milesPerGallon);

		System.out.print("Enter miles driven: ");
		milesDriven = input.nextDouble();
	}

	double combinedMilesPerGallon = totalMiles / totalGallons;

	System.out.print("The combined miles per gallon: " + combinedMilesPerGallon);
   }

}

File: Python-Assignment/chapterThree/question11.py
total_miles = 0
total_gallons = 0

miles_driven = int(input("Enter miles driven: "))

while miles_driven != -1:
	total_miles = total_miles + miles_driven

	gallons_used = float(input("Enter gallons used: "))
	total_gallons = total_gallons + gallons_used

	miles_per_gallons = miles_driven / gallons_used
	print("The miles per gallon for this week: ", miles_per_gallons)

	miles_driven = int(input("Enter miles driven: "))

combined_miles_per_gallons = total_miles / total_gallons

print("The combined miles per gallon: ", combined_miles_per_gallons)